web/src/main/java/org/mskcc/cbio/oncokb/controller/EvidenceController.java

```diff
@@ -222,6 +222,27 @@ ResponseEntity updateVUS(@ApiParam(value = "hugoSymbol", required = true) @PathV
         Set<ClinicalTrial> clinicalTrials = queryEvidence.getClinicalTrials();
         String propagation = queryEvidence.getPropagation();
         
+        List<String> cancerTypes = new ArrayList<>();
+        List<String> subTypes = new ArrayList<>();     
+        Boolean cancerEvidence = true;
+        if(cancerType == null) {
+            cancerEvidence = false;
+        } else {
+            cancerTypes = Arrays.asList(cancerType.split(","));
+            if(subType != null) {
+                subTypes = Arrays.asList(subType.split(","));
+                for(int i = 0;i < subTypes.size();i++) {
+                    if(subTypes.get(i).equals("null")) {
```

Comment:
If you manually inject the null in the frontend, can we just inject empty string? It will make the request size smaller(very minor)